namespaces/non-production.k8s.integration.dsd.io/formbuilder-platform-staging/formbuilder-platform-staging-admin-role.yaml

```diff
@@ -0,0 +1,64 @@
+# Role to allow listing of namespaces.
```

Comment:
The `-namespaces-list` and `-basic-user` `ClusterRole` and `ClusterRoleBindings` are not needed here, they are already applied for all users on the cluster level.